todo.sh

```diff
@@ -3,60 +3,63 @@
 # To be able to separate the code into several files after creating a symlink
 # to ~/bin/todo the path to the source root is good to have.
 # The relative paths to the project files can now  be used
-# like so $lib_dir/someProjectScript.sh
+# like so $this_dir/someProjectScript.sh
 this_path=${BASH_SOURCE[0]}
-lib_dir="$(dirname "$this_path")"
+this_dir="$(dirname "$this_path")"
+this_file="$(basename "$this_path")"
+
+
+for src in *.sh; do   
+    if [ "$src" = "$this_file" ]; then
+        continue
+    fi
+
+    [ -f "$src" ] && . "$src"
```

Comment:
You have to source the absolute path. If you're not in the repository directory and run the generated script (which runs with absolut path), it won't be able to find the files.

Try `[ -f "$this_dir/$src" ] && . "$this_dir/$src"`

Reply:
Right!

Reply:
I think the versioin now is right beacuse we need to list the files in the absolute path right? Otherwise we wont find the files in the first place i.e. `for src in "$this_dir/"*.sh; do`